### DISCUSSION TITLE
<Service> | "Links to the main resources": How to configure (turn off) the comma separated list in parentheses?

### DISCUSSION THREAD

**Discussion by Sverre-Iversen** [ldproxy/ldproxy#1170] [Configuration]
Usually the the comma separated list in parentheses is not shown, but sometimes ldproxy shows it. Sometimes this list is too long and unreadable. Is it possible to turn it off?

![Screenshot 2024-02-19 103401](https://github.com/interactive-instruments/ldproxy/assets/1905113/eb919d2e-8786-46fd-88c6-0668a4f3bd7c)

**Comment by cportele** [comment]
The list is always provided, if there are four collections or less. I agree, it would be better to have an option to explicitly control the behavior.
